packages/Sandblocks-Core/SBEditor.class.st

```diff
@@ -1280,6 +1322,12 @@ SBEditor >> removeHelperTimer [
 	self removeAlarm: #showHelper
 ]
 
+{ #category : #'as yet unclassified' }
+SBEditor >> removeIndicatorFor: aMorph [
+	
+	scroll removeIndicatorFor: aMorph
```

Comment:
I know that the law of demeter would ask you to do add a method like this, my personal preference would argue for not duplicating it in SBEditor though. SBEditor already has a very large interface and anything that can be deferred to a helper object (such as the overlay/scroll) is probably a good thing :D Or do you have another motivation for moving it here?